### DISCUSSION TITLE
imtoken嵌套的h5如何怎么调用app的保存图片的方法呢

### DISCUSSION THREAD

**Discussion by boonook** [consenlabs/webview#41] [Q&A] [answered]
imtoken嵌套的h5如何怎么调用app的保存图片的方法呢

**Comment by unix** [comment, marked as answer]
没有对外提供保存图片 API (涉及用户终端的权限)，但可以 [分享图片](https://imtoken.gitbook.io/developers/products/webview/sdk#share)。

**Reply by boonook** [reply]
好的非常感谢
